sdk/src/main/java/io/opentelemetry/sdk/trace/RecordEventsReadableSpan.java

```diff
@@ -368,16 +368,34 @@ public void addEvent(String name) {
     addTimedEvent(TimedEvent.create(clock.nowNanos(), name));
   }
 
+  // TODO: Use timestamp.
```

Comment:
can you create an issue to track this TODO?

Reply:
https://github.com/open-telemetry/opentelemetry-java/issues/633